mods/ra/maps/allies-01/map.yaml

```diff
@@ -488,12 +487,17 @@ Rules:
 		Cargo:
 			Types: Einstein
 			MaxWeight: 1
+		AutoSelectionSize:
+		RenderSprites:
+			Image: tran
 	TRAN.Insertion:
 		Inherits: TRAN.Extraction
-		RenderUnit:
-			Image: tran
+		WithFacingSpriteBody:
 		Cargo:
 			MaxWeight: 0
+		AutoSelectionSize:
+		RenderSprites:
+			Image: tran
```

Comment:
These can probably be removed, as they are inherited from `TRAN.Extraction`. It's worth to check about `RenderSprites` though.